gatsby/src/utils/index.ts

```diff
@@ -0,0 +1,15 @@
+export * from "./types";
+
+/**
+ * Returns whether the given object is truthy or not bus casted to the object's type.
```

Comment:
Unbelievably pedantic I know, but is this a typo? Assuming "bus" should be "but"